src/js/node/worker_threads.ts

```diff
@@ -135,9 +136,7 @@ function fakeParentPort() {
   });
 
   Object.defineProperty(fake, "close", {
-    value() {
-      return process.exit(0);
```

Comment:
`process.exit` in a `Worker` does not crash the process. It terminates the Worker and sets the exit code accordingly, but does not exit

Reply:
The worker termination, clean or not, is not the expected behavior at worker_threads parent port closing.

Reply:
only observable behavior i can find in closing the parent port is simply being unable to send more messages from it.

we can make our emulation better by simply having an internal flag `let closed = false` and have close set it to true, and prevent messages. but tbh this is good enough.

Reply:
I think the parent port emulation in node:worker_threads design needs thoughts. And I unfortunately does not enough knowledge on bun internals to propose a scalable message port implementation (unlike the node.js one) and a complete emulation for node:worker_threads compatibility bun module. At least for now.